app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -3,24 +3,36 @@
 miqHttpInject(angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap', 'ManageIQ']))
 .controller('containerTopologyController', ContainerTopologyCtrl);
 
-ContainerTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService'];
+ContainerTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService', '$window'];
+
+function ContainerTopologyCtrl($scope, $http, $interval, $location, topologyService, $window) {
+  $(function($) {
```

Comment:
`$(function() { ... })` is equivalent to `$(document).ready(function() { ... })` .. so I wonder why you're using both :) especially since it can't never be not ready if you're already in a controller?